docs/features/optin.md

```diff
@@ -0,0 +1,56 @@
+# Opt in to Use Alpha Version Controllers
+
+> `Feature State`: `alpha` as of version v1.125
```

Comment:
IIRC, this feature is enabled in 1.124.0. Could you double check?

Reply:
Would prefer to specify 1.125 here, so the user does not enable direct controller for sqlinstance on 1.124 (which does not have fixes from this last iteratino).

Reply:
It worths noting that we already revealed this annotation in [v1.123.1](https://github.com/GoogleCloudPlatform/k8s-config-connector/releases/tag/v1.123.1) for DataflowFlexTemplateJob resource. So this mismatch could confuse users.

Reply:
Ok, I updated it to say v1.123.1. I am still worried that the users with sqlinstance issues will interpret this to mean their problems will be solved by enabling the direct controller annotation for sqlinstance resources in v1.123.1 / v1.124.0 (which have known bugs for sqlinstance direct controller).

Reply:
Understood. How about we give a more dedicated, per-resource release version guide, like sqlinstance 1.125, dataflowflexTempalteJob 1.123.1, ...and we can also get rid of the long explanation about how to find the right versions to apply this labels.

Reply:
That could work also. Though, I'd rather not have to maintain such a list. Hopefully we will soon switch over to direct only controller for dataflowflextemplatejob / sqlinstance. At that point, we'd have to update this doc again.

Reply:
Maybe the concern in https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/3091#discussion_r1828560321 is ok -- it is experimental code after all.